src/graphql/schema/Trip/resolvers.js

```diff
@@ -108,6 +112,50 @@ export default {
       const newTrip = unlockBudget(tripID, user, Trip);
       pubsub.publish('TRIPINFO_CHANGED', { tripInfoChanged: newTrip });
       return newTrip;
+    },
+    addGeneralMessage: async (_, { tripID, message }, { Trip, user }) => {
+      const newTrip = await writeGeneralMessage(tripID, message, user, Trip);
```

Comment:
For consistency, might I suggest to rename newTrip to updatedTrip in the added resolvers?